lib/puppet/network/http/request.rb

```diff
@@ -19,39 +19,42 @@ def route_into(prefix)
     self.class.new(headers, params, method, path, routing_path.sub(prefix, ''), client_cert, body)
   end
 
-  def format
+  def formatter
```

Comment:
yep, unfortunately puppet uses both format and formatter to refer to a `Format` object, and other times `format` refers to either the mime type or the registered name, e.g. `json`.